Assets/Scripts/StarComponent.cs

```diff
@@ -1,10 +1,10 @@
-﻿using UnityEngine;
+﻿using UnityEngine;
 using UnityEngine.EventSystems;
 
 public class StarComponent : MonoBehaviour
 {
     public Star starData;
-    public Color starColor;
+    public Color starColor;

```

Comment:
Looks like editors are fighting.  This only had a line feed no carriage return, so VS Code changed the line ending from LF to CRLF.

Reply:
Same is true of some of the other changes in the file.  VS Code is converting LF to CRLF.